Trellix/trellix-network-security/tests/infection_match.json

```
@@ -0,0 +1,65 @@
+  },
+  "expected": {
+    "message": "CEF:0|Trellix|MPS|10.0.0.992057|IM|infection-match|1|spt=1046 smac=6c:af:1a:fb:fe:a7 dpt=80 dmac=00:78:db:db:96:f6 dvchost=cms-nx5600-1.eng.fireeye.com dvc=3.4.5.6 cn1Label=vlan cn1=0 cn2Label=sid cn2=607378 cn3Label=cncPort cn3=80 cs1Label=sname cs1=Local.Infection cs4Label=link cs4=https://cms-nx5600-1.eng.fireeye.com/detection/objects?uuid\\=2cededd4-cb4b-42de-9d7e-8e1ce56a9fe6 cs5Label=cncHost cs5=2011::1:6377:90aa cs6Label=channel cs6=GET /m/web.php HTTP/1.1::~~Host: zebrabel1.co.cc::~~User-Agent: Mozilla/5.0 (Windows; U; Windows NT 5.1; en-US; rv:1.9b5) Gecko/2008032620 Firefox/3.0b5::~~Accept: text/html,application/xhtml+xml,application/xml;q\\=0.9,*/*;q\\=0.8::~~Accept-Language: en-us,en;q\\=0.5::~~Accept-Encoding: gzip,deflate::~~Accept-Charset: ISO-8859-1,utf-8;q\\=0.7,*;q\\=0.7::~~Keep-Alive: 300::~~Connection: keep-alive::~~Referer: http://zebrabel1.co.cc/m/::~~::~~ proto=tcp rt=Sep 05 2023 16:28:55 UTC externalId=20966332 act=notified c6a3=1c83:125e:807c:d317:d732:d30b:6af0:d34f c6a3Label=Attacker IP c6a2=decc:4ab1:133a:f9ce:18d2:7c83:2142:601e c6a2Label=Victim IP requestMethod=GET requestClientApplication=Mozilla/5.0 (Windows; U; Windows NT 5.1; en-US; rv:1.9b5) Gecko/2008032620 Firefox/3.0b5 requestContext=http://zebrabel1.co.cc/m/ devicePayloadId=2cededd4-cb4b-42de-9d7e-8e1ce56a9fe6 start=Sep 05 2023 16:28:55 UTC dvcmac=e3:e9:d0:5e:ba:8e",
+    "event": {
+      "kind": "event",
+      "dataset": "infection-match",
+      "severity": 1,
+      "start": "2023-09-05T16:28:55Z",
+      "action": "notified",
+      "url": "https://cms-nx5600-1.eng.fireeye.com/detection/objects?uuid\\=2cededd4-cb4b-42de-9d7e-8e1ce56a9fe6",
+      "category": ["network"],
```

Could you update the parser to categorize `infection-match` event as `intrusion_detection`?
```suggestion
      "category": ["intrusion_detection"],
```